src/common/cookie.cc

```diff
@@ -103,6 +105,8 @@ Cookie::Cookie(std::string name, std::string value)
 Cookie
 Cookie::fromRaw(const char* str, size_t len)
 {
+    std::cout<<"From Raw initiated with" << str <<" " << endl;
```

Comment:
Please be consistent with the existing coding style, and consistent within your own line of code.  The "<<" operator should have a space on either side of it.